documentation/asciidoc/stories/assembly_setting_up_xsite.adoc

```diff
@@ -2,18 +2,44 @@ ifdef::context[:parent-context: {context}]
 [id='setting-up-xsite']
 :context: cross-site
 = Setting up cross-site replication
-Cross-site replication allows you to back up data from one {brandname} cluster to another.
+Ensure service availability, and guarantee business continuity, with {ispn_operator} by configuring cross-site replication to back up data between {brandname} clusters.
```

Comment:
"guarantee business continuity" I think we should remove this as it sounds like marketing, not technical docs.